Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM centos:6
```

Comment:
Could do with a short readme at the top of the file to explain what this file is for and why it's here.